src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/documentcatergory/UploadGeneralDocumentsCategoriser.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.documentcatergory;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.finrem.caseorchestration.helper.DocumentHelper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseDocument;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseData;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.UploadGeneralDocument;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.UploadGeneralDocumentType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.document.DocumentCategory;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.FeatureToggleService;
+
+@Configuration
+public class UploadGeneralDocumentsCategoriser extends DocumentCategoriser {
+
+    private final DocumentHelper documentHelper;
+
+    public UploadGeneralDocumentsCategoriser(FeatureToggleService featureToggleService, DocumentHelper documentHelper) {
+        super(featureToggleService);
+        this.documentHelper = documentHelper;
+    }
+
+    @Override
+    protected void categoriseDocuments(FinremCaseData finremCaseData) {
+
+        if (CollectionUtils.isNotEmpty(finremCaseData.getUploadGeneralDocuments())) {
+            finremCaseData.getUploadGeneralDocuments().forEach(doc -> checkTypeAndSetCategory(doc.getValue()));
+        }
+
+    }
+
+    private void checkTypeAndSetCategory(UploadGeneralDocument document) {
```

Comment:
Minor and not a reason to not approve but perhaps this could have been made more readable e.g. importing the constants or adding them to a list so you could just do something like 
`if (APPLICANT_DOC_TYPES.contains(document.getdocumentType())...`

Reply:
Thanks, updated